### DISCUSSION TITLE
PaddleOCR/en/ppstructure/quick_start

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#13953] [Q&A]
# PaddleOCR/en/ppstructure/quick_start

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/en/ppstructure/quick_start.html

**Comment by komalpasumarthy** [comment]
when save_structure_res is saving the data in csv why can't the data be loaded into dataframe again? Maybe because of encoding? how to save the table into a csv using utf-8 encoding? is there any other alternative? or is it because of any other reasons? can you clarify my query please???
